.changelog/261.breaking.md

```diff
@@ -0,0 +1,8 @@
+bring Rosetta dependencies up to date
+
+We're updating our Rosetta dependencies, including the
+Go SDK, the CLI, and along with those, the Rosetta API
+specifications.
+
+Changes to the specifications may have introduced
```

Comment:
Should we perhaps tune a bit and say something like:

Updating the Rosetta API to version `<NEW-VERSION>` along with its Go SDK is a significant update and may have introduced breaking changes.

?

Reply:
yeah sure

there's actually no major version bump in their specs https://github.com/coinbase/rosetta-specifications/releases

but plenty of topmost-nonzero version number changes in the SDK and CLI, so still